README.md

```diff
@@ -1,4 +1,6 @@
-# React Storybook Chapters Addon
+# ⛔ [DEPRECATED] React Storybook Chapters Addon
+
+## This addon is no longer maintained and is now considered deprecated. We recommend Storybook version 6 be used instead.
```

Comment:
This is a good heading, but it might be worth also including an explanation about why Storybook 6 replaces the need for this addon. Maybe something like what the other chapters addon is [doing](https://github.com/sm-react/storybook-chapters#this-addon-no-longer-maintained-and-considered-as-deprecated-in-favor-of-storybook-native-nesting-nevertheless-if-youre-interested-to-continue-to-maintain-this-project-get-in-touch-with-me-please), minus the part about getting in touch with us.

Reply:
And perhaps link to the storybook documentation about their native grouping and hierarchy: https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy